rca/project_styleguide/templates/patterns/pages/editorial/editorial_detail.html

```diff
@@ -25,9 +25,7 @@
                     <nav class="title-area__breadcrumb" aria-label="Breadcrumb">
                         {% include "patterns/molecules/breadcrumb/breadcrumb.html" %}
                     </nav>
-                    {% if page.title %}
-                        <h1 class="title-area__heading heading heading--display-two">{{ page.title }}</h1>
-                    {% endif %}
+                    <h1 class="title-area__heading heading heading--display-two" id="main-content">{{ page.title }}</h1>
```

Comment:
i'm a bit out of the loop with the structure of the site now but I presume moving `id="main-content"` doesn't mean that there's a chance that there will be more than 1 of them on the page?

Reply:
i'm guessing not as they're going on the `<h1>`'s...

Reply:
yea, only pages that have multiple h1's are ones with modals on, which wouldn't need the anchor on anyhow, so think we're good. Does feel annoying that we need to move this id into the templates instead of using base, but needs must…